src/jupyter_contrib_nbextensions/nbextensions/contrib_nbextensions_help_item/README.md

```diff
@@ -0,0 +1 @@
+The contrib_nbextensions_help_item is a tiny extension that just adds an item in the notebook's help menu, pointing to the jupyter_contrib_nbextensions at readthedocs.
```

Comment:
Actually, to play nicely with our new auto-generating documentation, this readme should have a single heading with the name of the nbextension, so that it gets into tables of contents correctly, something like

    help menu entry
    ===============

    The contrib_nbextensions_help_item is ...